split.js

```diff
@@ -143,6 +156,7 @@ Split = function (ids, options) {
             pair.gutter = gutter;
         }
 
-        el.style.width = 'calc(' + options.widths[i] + '% - ' + options.gutterWidth / 2 + 'px)';
```

Comment:
Maybe the simplest fix would be to adopt a different strategy for allowing for the gutter widths: instead of subtracting half-a-gutter from each end pane, and a full gutter from middle pane(s?), maybe subtract a **full** gutter from *every* pane *except the first*?

One advantage of the suggested alternative is that it would avoid having to divide the gutter width in half, which seems awkward when the gutter width is an odd number.